pkg/util/log/log.go

```diff
@@ -164,10 +163,7 @@ func newPrometheusLogger(l dslog.Level, format dslog.Format, reg prometheus.Regi
 		writer = log.NewSyncWriter(writer)
 	}
 
-	baseLogger := log.NewLogfmtLogger(writer)
-	if format.String() == "json" {
-		baseLogger = log.NewJSONLogger(writer)
-	}
+	baseLogger := dslog.NewGoKitWithWriter(format, writer)
```

Comment:
Is this JSON by default?

Reply:
@dannykopping no, [by default](https://github.com/grafana/dskit/blob/main/log/gokit.go#L48-L51) it's `logfmt`. It should be the same behaviour as before, no?

Reply:
I misread 👍